bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -96,6 +101,22 @@ public void removedService(final ServiceReference reference, final Object servic
             }
         };
         thingRegistryServiceTracker.open();
+        linkRegistryServiceTracker = new ServiceTracker(this.bundleContext, ItemChannelLinkRegistry.class.getName(),
```

Comment:
Sorry for the question at all. It seems I missed that this is the BaseThingHandler implementation (also after you have written "ThingHandlers"). :wink: long days ATM